tests/test_parallel_execution.py

```diff
@@ -34,14 +34,16 @@ class TestParallelExecution(unittest.TestCase):
 
     @expect_failure_if_no_gpu
     def test_single_gpu(self):
-        net = torch.nn.Conv2d(1, 1, 3, padding=1)
+        device = torch.device("cuda:0")
```

Comment:
```suggestion
        device = torch.device("cuda")
```

Reply:
Yes, it's better to use "cuda", will update when we enable distributed training in CI later.
We are now code-freezing for v0.3 release.
Thanks.